src/components/calculator.js

```diff
@@ -4,26 +4,26 @@ const buttonsGroup = require("./buttons-group");
 function calculator(id) {
 	const buttonsTop = buttonsGroup("buttons-top", [
 		button("C","clear-all upper left"),
-		button("%","bottom"),
+		button("+/-","less-more upper"),
 		button("CE","del upper")
 	]);
 	const buttonsMiddle = buttonsGroup("buttons-middle",[
-		button("1"), button("2"),button("3"),
-		button("4"), button("5"),button("6"),
-		button("7"), button("8"),button("9"),
+		button("1","left"), button("2"),button("3"),
+		button("4","left"), button("5"),button("6"),
+		button("7","left"), button("8"),button("9"),
 	]);
 	const buttonsRight = buttonsGroup("buttons-right",[
+		button("%","bottom upper"),
 		button("/","right"),
 		button("*","right"),
 		button("+","right"),
 		button("-","bottom right"),
-		button("=","equal bottom"),
 	]);
 
 	const buttonsBottom = buttonsGroup("buttons-bottom",[
 		button("0","bottom"),
```

Comment:
the same here, why do you need the bottom class/id for those buttons?

Reply:
and why do you need the `equal` identifier?

Reply:
this one i forgot to delete, because i don't need this anymore

Reply:
please review your PR and see what you need and what you don't need!

Reply:
ok , when was pushing i forgot about review it by myself. I need to familiarize with this :p